client.go

```diff
@@ -190,6 +190,26 @@ func (c *Client) trigger(channels []string, event string, data interface{}, sock
 	return unmarshalledBufferedEvents(response)
 }
 
+type BatchRequest struct {
+	Batch []Event `json:"batch"`
+}
+
+func (c *Client) TriggerBatch(batch []Event) (*BufferedEvents, error) {
+	payload, err := json.Marshal(&BatchRequest{batch})
+	if err != nil {
+		return nil, err
+	}
+
+	path := fmt.Sprintf("/apps/%s/batch_events", c.AppId)
+	u := createRequestURL("POST", c.Host, path, c.Key, c.Secret, authTimestamp(), c.Secure, payload, nil, c.Cluster)
```

Comment:
yeah - i had only known go for like 3 days before writing the library. i'd probably do it a lot cleaner if i did it now